phys/module_diag_nwp.F

```diff
@@ -455,6 +460,39 @@ SUBROUTINE diagnostic_output_nwp(                                  &
 !  !$OMP END PARALLEL DO
    ENDIF
 
+   IF (PRESENT(qvolg_curr)) THEN
+   WRITE(outstring,*) 'GT-Debug, this mp scheme, ', mphysics_opt, ' has graupel volume mixing ratio'
+   CALL wrf_debug (150, TRIM(outstring))
```

Comment:
The NSSL schemes also have qvolg. Would they give a reasonable diagnostic or should this be limited to Thompson?

Reply:
It should be limited to Thompson, @dudhia. That's because the diagnostic assumes there are 9 possible densities and it's unlikely NSSL makes the same assumptions. Should I add a Case (THOMPSONGH) statement here?

Reply:
@gthompsnWRF would you be able to confirm if this is necessary, or should this work with NSSL too?